.circleci/continue_config.yml

```diff
@@ -516,6 +516,23 @@ jobs:
 workflows:
   version: 2.1
 
+  lint_docs_only:
```

Comment:
This might be slightly misleading, because afaik the linting jobs run on both the code and docs. So this should probably just be `lint_only`